tests/plugins/test_flatpak_create_dockerfile.py

```diff
@@ -0,0 +1,124 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from flexmock import flexmock
+
+import json
+import responses
+import os
+import pytest
+from urlparse import urlparse, parse_qs
+
+from atomic_reactor.inner import DockerBuildWorkflow
+from atomic_reactor.plugins.pre_flatpak_create_dockerfile import FlatpakCreateDockerfilePlugin
+from atomic_reactor.plugin import PreBuildPluginsRunner
+from atomic_reactor.source import VcsInfo
+from atomic_reactor.util import ImageName
+
+from tests.constants import (MOCK_SOURCE, FLATPAK_GIT, FLATPAK_SHA1)
+from tests.fixtures import docker_tasker
```

Comment:
Stickler CI doesn't know about py.test fixtures. Lets add `# noqa` to every line it complains about unused fixture (or redefinition of unused fixture)